website/content/docs/install/manual-bootstrap.mdx

```diff
@@ -1,11 +1,8 @@
 ---
 layout: docs
-page_title: Manual Bootstrapping
+page_title: Manually Bootstrap a Datacenter
 description: >-
-  When deploying Consul to a datacenter for the first time, there is an initial
-  bootstrapping that must be done. As of Consul 0.4, an automatic bootstrapping
-  is available and is the recommended approach. However, older versions only
-  support a manual bootstrap that is documented here.
+Manually bootstrap a datacenter to deploy your Consul servers and join them together for the first time. As of Consul v0.4, automatic bootstrapping is recommended instead.
```

Comment:
I didn't even know this page existed. It turns out it's a hidden page ... which makes sense: why would we promote a page that hasn't been relevant since Consul 0.3? :P

Reply:
Per @jkirschner-hashicorp's comment, 0.4 turned eight years old this month. Should we retire this page? I don't think we have a policy w/r/t how long we need to keep docs around.